cosmos/providers/databricks/workflow.py

```diff
@@ -244,6 +251,7 @@ def __init__(
         python_params: list = None,
         spark_submit_params: list = None,
         max_concurrent_runs: int = 1,
+        extra_job_params: dict = None,
```

Comment:
nit: can we type this stronger?
```suggestion
        extra_job_params: dict[str, Any] = None,
```